pages/desktop/page_provider.py

```diff
@@ -57,14 +57,14 @@ def questions_page(self, username=None, password=None):
         from pages.desktop.questions_page import QuestionsPage
         return self._go_to_page(QuestionsPage(self.testsetup), username, password)
 
-    def search_page(self, username=None, password=None):
-        from pages.desktop.search_page import SearchPage
-        return self._go_to_page(SearchPage(self.testsetup), username, password)
-
     def refine_search_page(self, username=None, password=None):
         from pages.desktop.refine_search_page import RefineSearchPage
         return self._go_to_page(RefineSearchPage(self.testsetup), username, password)
 
+    def search_page(self, username=None, password=None):
```

Comment:
+ 1 to alphabetical ordering of methods, but -1 to making an unnecessary change in a file that is otherwise untouched. I think it's fine to leave it as is, as I do like the change, but generally we try to avoid touching unrelated files when working on a specific issue, as it complicates the review process and can potentially add bugs.